File: app/src/main/java/com/github/shadowsocks/database/SSRSub.java
package com.github.shadowsocks.database;


import com.j256.ormlite.field.DatabaseField;

public class SSRSub {
    @DatabaseField(generatedId = true)
    public int id = 0;

    @DatabaseField
    public String url = "";

    @DatabaseField
    public String url_group = "";
}

File: app/src/main/java/com/bige0/shadowsocksr/database/SSRSub.kt
package com.bige0.shadowsocksr.database

import com.j256.ormlite.field.*

class SSRSub
{
	@DatabaseField(generatedId = true)
	var id = 0

	@DatabaseField
	var url = ""

	@DatabaseField
	var url_group = ""
}
